### DISCUSSION TITLE
API Documentation

### DISCUSSION THREAD

**Discussion by brianjz** [vladmandic/sdnext#743] [General]
Now that the SD/Gradio API page (at /docs) is gone and the new option (?view=api) is completely busted. Does anyone have a copy/screenshot/PDF of the documentation? I was messing around with it before the change and now I can't get back to the list of endpoints without installing an older version. Thanks.

**Comment by vladmandic** [comment]
i've created a monkey-patch to restore /docs, will publish on monday.

**Reply by jeffrey2212** [reply]
Hi, May I know the /docs is restored yet? Thanks!

**Reply by vladmandic** [reply]
yes for over a month, but you need to use `--docs` cmdflag as i don't mount it by default, simply because its needed by very small subset of users.

**Reply by jeffrey2212** [reply]
Cool, I can access to the doc now. Thanks!
